src/index.html

```diff
@@ -10,23 +10,34 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Airbnb Search Bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body>
+    <form
+      class="search-bar"
+      data-qa="big"
+    >
+      <input
+        class="search-bar__input"
+        data-qa="keypress"
+        type="text"
+        placeholder="Try “Los Angeles”"
+      />
+    </form>
+    <form
+      class="search-bar"
+      data-qa="small"
+    >
+      <input
+        class="search-bar__input search-bar__input--small"
+        type="text"
+        placeholder="Try “Los Angeles”"
```

Comment:
Keep your attributes correctly formatted. Since the `input` element has more than two attributes, each attribute should start on a new line with 2-space indentation relative to the tag.